### PR TITLE
Update to latest vscode-languageclient

### PR DESCRIPTION
**Pull request by Colengms**

### CODE REVIEW

File: Extension/src/LanguageServer/protocolFilter.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ export function createProtocolFilter(clients: ClientCollection): Middleware {
                 // didOpen), and first becomes visible.
             }
         },
-        didChange: (textDocumentChangeEvent, sendMessage) => {
+        didChange: async (textDocumentChangeEvent, sendMessage) => {
```
Comment (bobbrow) [inline Extension/src/LanguageServer/protocolFilter.ts:77]: Functions should only be marked `async` if they use `await` somewhere in the body.  Does this change the return type from `void` to `Promise<void>` without an `await`?

Comment (Colengms) [reply Extension/src/LanguageServer/protocolFilter.ts:77]: Yes, this changes the return type from `void` to a `Promise<void>` automatically, and resolves the compilation error.  Alternatively, I could return a resolved `Promise<void>`, but this seems cleaner.

**Comment by sean-mcmanus** [comment]
What milestone/release is this targetting?

**Comment by bobbrow** [comment]
This can wait.

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**